apps/api/src/sibyl/crawler/graph_integration.py

```
@@ -913,3 +915,4 @@ async def integrate_document_with_graph(
         return IntegrationStats()
 
     return await service.process_chunks(chunks, source_name)
+    DEFAULT_MAX_TOKENS = 1000
```

Ruff (0.15.13)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)